src/controllers/queries/__tests__/helpers.ts

```diff
@@ -13,13 +15,32 @@ interface Query {
   updated_at: Date
   status: QueryStatus
 }
+
+//this is not the most elegant way to have the sample data
+//(it is being poppped from the end every time a get is called on a db)
+const sampleArray = [
+  [{ company_name: 'VER123', status: 'pending', id: '11' }],
+  [{ company_name: 'VER123', status: 'pending', id: '11' }],
+  [{ id: 'x', status: 'xx', connection_id: '11' }],
+  [{ company_name: 'VER123', status: 'pending', id: '11' }],
+  [{ id: 'x', status: 'xx', connection_id: '11' }],
+  [{ company_name: 'bar', status: 'pending', id: '11' }],
+]
```

Comment:
you might be able to do something like

```
const sampleArray = {
  query: [{ id: 'x', status: 'xx', connection_id: '11' }],
  connection: [{ company_name: 'VER123', status: 'pending', id: '11' }],
}


  const dbMock = {
    get: (model: string) => {
      return Promise.resolve(sampleArray[model])
    },
  } as unknown as Database
```
but need a way to handle the two different connections that are returned